src/components/Login.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import './Login.css';
+import { Button, Card, Space } from 'antd';
+import { GoogleCircleFilled } from '@ant-design/icons';
+import firebase, { auth } from './fire';
+
+export class Login extends React.Component {
+   
+    render() {
+        const googleSignIn = e => {
+            var provider = new firebase.auth.GoogleAuthProvider();
```

Comment:
lets put `const googleProvider = new firebase.auth.GoogleAuthProvider()` in `FirebaseService` then we import that here and use it